agent/agentbootstrap/bootstrap.go

```diff
@@ -501,7 +501,7 @@ func initControllerCloudService(
 		// this should never happen.
 		return errors.Errorf("environ %T does not implement ServiceManager interface", env)
 	}
-	svc, err := broker.GetService(k8sprovider.JujuControllerStackName, caas.ModeWorkload, true)
+	svc, err := broker.GetService(k8sconstants.JujuControllerStackName, caas.ModeWorkload, true)
```

Comment:
I think agent/agentbootstrap should trend towards not being dependent on the Kubernetes provider, but this move is a step in the right direction

Reply:
I agree. This can be addressed in next iterations.